portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,35 @@
-#content {
+
+body {
+    position: relative;
+}
+.affix {
+    top: 20px;
+    z-index: 9999 !important;
+}
+div.col-sm-9 div {
+    height: 250px;
+    font-size: 28px;
+}
+@media screen and (max-width: 810px) {
+    #education, #work, #projects, #service, #interets, #family, #contact  {
+        margin-left: 150px;
```

Comment:
Consider using relative units like `rem` and `em` so your page scales with font size for accessibility.